UX/ui-service/MyOrders/design.md

```diff
@@ -0,0 +1,44 @@
+# My Orders
+
+- My Orders is a primary navigation item
+- Administrators
+  - View and approve or deny orders for other users
+- End Users
+  - View status of placed orders
+
+## Admin user
+
+### Review Orders
+![Image Title/Mockup Name](img/Order_01.png)
+- The Admin User sees all orders that have been placed.
+
+### Approve Order
+![Image Title/Mockup Name](img/Order_02.png)
+- The Administrator can check and approve an order or its component pieces
+
```

Comment:
@cshinn I don't think you meant to have the warning icons still showing here. Should these be the clock icon now too?